bin/bootstrax

```diff
@@ -126,7 +137,9 @@ timeouts = {
 }
 
 # Fields in the run docs that bootstrax uses
-bootstrax_projection = 'name start end number bootstrax data.host data.type data.location ini.processing_threads'.split()
+bootstrax_projection = f"name start end number bootstrax " \
```

Comment:
You're not using the f-string functionality here: it's not harmful but not needed either.